eva-remapping-ingest/src/main/resources/application.properties

```diff
@@ -0,0 +1,20 @@
+spring.batch.job.names=INGEST_REMAPPED_VARIANTS_FROM_VCF_JOB
```

Comment:
Does it need to be called `application.properties` ?

Reply:
It does not need to be called `application.properties` but that is the default file Spring looks for. If we want to use a different name it is possible but we will need to specify it via environment properties `spring.config.name` or `spring.config.location`. See [here](https://docs.spring.io/spring-boot/docs/current/reference/html/spring-boot-features.html#boot-features-external-config-files) for reference.